File: backend/Dtos/request/DietaRequestDTO.cs
namespace backend.Dtos.request;

public record DietaRequestDTO(
int Id_Paciente = 0,
    int Id_Nutricionista = 0,
    string Nombre = "",
    string Descripcion = "",
    DateTime Fecha_Inicio = default,
    DateTime Fecha_Fin = default,
    List<DietaComidaRequestDTO> Comidas = null!
);

File: frontend/src/types/dto/request/DietaRequestDTO.ts
import type { DietaComidaRequestDTO } from './DietaComidaRequestDTO';

export interface DietaRequestDTO {
    id_Paciente: number;
    id_Nutricionista: number;
    nombre: string;
    descripcion: string;
    fecha_Inicio: string;
    fecha_Fin: string;
    activa: boolean;
    comidas: DietaComidaRequestDTO[];
}